### PR TITLE
Update region-grower dependencies

### PR DESCRIPTION
**Pull request by adrien-berchet**
Update region-grower and dependencies to latest versions.

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-neuroc/package.py [hunk 1]
```diff
@@ -15,6 +15,7 @@ class PyNeuroc(PythonPackage):
     git = "ssh://bbpcode.epfl.ch/nse/NeuroC"
 
     version('develop', branch='master')
+    version('0.2.7', tag='neuroc-v0.2.7')
     version('0.2.6', tag='neuroc-v0.2.6')
```
Comment (arnaudon) [inline var/spack/repos/builtin/packages/py-neuroc/package.py:19]:
```suggestion

```

**Review by arnaudon — COMMENTED** [review]
you didn't change region-grower version?

File: var/spack/repos/builtin/packages/py-neuroc/package.py (hunk 1 above)
Comment (matz-e) [reply var/spack/repos/builtin/packages/py-neuroc/package.py:19]: What about 0.2.4? Could also always clean up later, 3 versions is quite a short history.

**Review by matz-e — APPROVED** [review]

File: var/spack/repos/builtin/packages/py-neuroc/package.py (hunk 1 above)
Comment (arnaudon) [reply var/spack/repos/builtin/packages/py-neuroc/package.py:19]: there is only one other package depending on that, but yeah, let's add this new one for now

**Review by arnaudon — APPROVED** [review]

File: var/spack/repos/builtin/packages/py-neuroc/package.py (hunk 1 above)
Comment (adrien-berchet) [reply var/spack/repos/builtin/packages/py-neuroc/package.py:19]: should I also add 0.2.4 then?

Comment (adrien-berchet) [reply var/spack/repos/builtin/packages/py-neuroc/package.py:19]: Ah nevermind, it's already there, sorry :innocent:

**Comment by adrien-berchet** [comment]
BlueBrain5 Deployment Test Build failed because region-grower==0.2.3 was not yet available on devpi. Is it possible to rerun only this job? Or should I rerun the entire pipeline?

**Pull request merged**